sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.netstandard2.0.cs

```diff
@@ -330,18 +334,21 @@ public partial class NetAppVolumeData : Azure.ResourceManager.Models.TrackedReso
         public int? CoolnessPeriod { get { throw null; } set { } }
         public string CreationToken { get { throw null; } set { } }
         public Azure.ResourceManager.NetApp.Models.NetAppVolumeDataProtection DataProtection { get { throw null; } set { } }
+        public System.Collections.Generic.IReadOnlyList<string> DataStoreResourceId { get { throw null; } }
```

Comment:
Does the elements of this array all resource identifiers as the name suggested? If so, please change the format of this property to `arm-id` so that it has proper type of `IReadOnlyList<ResourceIdentifier>`